spec/features/guest_sees_homepage_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+feature "visits homepage", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.